app/views/contacts/show.html.slim

```diff
@@ -0,0 +1,50 @@
+.row
+  .col-xs-12
+    h1= t_title(:show)
+
+.row
+  .col-xs-12
+    dl.dl-horizontal
+      - if kind == 'person'
+        dt= t_attr(:first_name)
+        dd= @contact.first_name
+
+        dt= t_attr(:last_name)
+        dd= @contact.last_name
+
+        dt= t_attr(:title)
+        dd= @contact.title
+
+        dt= t_attr(:gender)
+        dd= @contact.gender
+
+        dt= t_attr(:birthday)
+        dd= @contact.birthday
+
+        dt= t_attr(:birthplace)
+        dd= @contact.birthplace
+
+      dt= t_attr(:organisation)
+      dd= @contact.organisation
+
+      dt= t_attr(:org_role)
+      dd= @contact.org_role
+
+- if kind == 'person'
```

Comment:
> Acceptance Criteria:
> - Models: Contact, Address, ContactPoint
> .....

So If there was this scaffold in the story and we where talking about this beforhand, you're pissing me off pretty badly if you're comming allong now and tell me that I should not create a contact model and told chrisa to scaffold another contact model at the same time.
👎